staker/multi_protocol/multi_protocol_staker.go

```diff
@@ -107,13 +111,18 @@ func (m *MultiProtocolStaker) Initialize(ctx context.Context) error {
 		return err
 	}
 	if boldActive {
+		stakeTokenContract, err := m.l1Reader.Client().CodeAt(ctx, m.stakeTokenAddress, nil)
```

Comment:
Shouldn't this code be in setupBoldStaker? As-is I don't think it's run if bold is enabled after startup